test/Table_Tests/src/Database/SQLite_Spec.enso

```diff
@@ -118,6 +118,18 @@ sqlite_specific_spec prefix connection =
             expected_code = code_template.replace "{Tinfo}" tinfo
             t.distinct ["strs"] . to_sql . prepare . should_equal [expected_code, []]
 
+    Test.group prefix+"Math" <|
+        table_builder = setup.table_builder
+        do_round n dp=0 use_bankers=False =
+            table = table_builder [["x", [n]]]
+            result = table.at "x" . round dp use_bankers
+            result.to_vector.at 0
+
+        Test.specify "NaN/Inf" <|
+            do_round Number.nan . should_equal Nothing
+            do_round Number.positive_infinity . should_equal 9223372036854775807
+            do_round Number.negative_infinity . should_equal -9223372036854775808
```

Comment:
Should we document that the behaviour for these values differs by backend?

IMO it may be worth it

OTOH there are many subtle differences between DBs and we probably don't document most of them, so not sure if this one needs to be

Reply:
Good idea; I added docs.